Jim-Hodapp-Coaching/refactor-platform-rs
# Update SeaORM to latest stable version

## Description
This PR updates SeaORM to the latest stable version `1.1.0`. It required updating `axum-login` and other related dependencies as well.


#### GitHub Issue: None

### Changes
* `SeaORM` to `v1.1.0`
* `axum-login` to `v0.16.0`
* `sqlx` and `sqlx-lite` to `0.8.2`
* Small updates to how types are imported based on changes in `tower-sessions`/`axum-login`

### Testing Strategy
1. Make sure `cargo test` passes
2. Run through all current frontend functionality on branch `main`


### Concerns
None
